shaders/lang/ru_RU.lang

```diff
@@ -1,46 +1,105 @@
-option.sunPathRotation = Угол наклона солнца/луны
-option.sun_illuminance = Свет Солнца
-option.moon_illuminance = Свет Луны
-option.MOONPHASE_BASED_MOONLIGHT = Уровень яркости фаз луны
-option.TRANSLUCENT_COLORED_SHADOWS = Полупрозрачные цветные тени
-option.OLD_LIGHTLEAK_FIX = Исправление утечки света
-
-option.RESOURCEPACK_SKY = Небо текстурпака
-    value.RESOURCEPACK_SKY.0 = Выкл.
-    value.RESOURCEPACK_SKY.1 = Небо + Шейдер солнца/луны
-    value.RESOURCEPACK_SKY.2 = Все небо
-
-screen.Waving_Stuff = Колыхание растительности
-    option.WAVY_PLANTS = Колыхание растительности
-    option.WAVY_STRENGTH = Интенсивность колыхания
-    option.WAVY_SPEED = Скорость колыхания
-
-screen.Direct_Light = Прямой свет
+ ____________________
+|                    |
+| By ShrikerChestTree|
+|   Version: 1.1.0   |
+|____________________|
+
+{~=~=~BLISS SHADERS~=~=~}
+
+option.SHADER_VERSION_LABEL = Bliss В-Разраб(In-Dev) Число Коммита
+    prefix.SHADER_VERSION_LABEL = КОММИТ #
+
+# если вы здесь, чтобы переводить, это шутка, они ничего не выполняют того, что говорят.
+# Okay)
+profile.QUALITY=Качество
+profile.PERFORMANCE=Производительность
+
+{#####_Misc. Settings_#####}
+
+screen.Misc_Settings = Настройка §cОтладочный§r/Экспериментальный
+
+|/////Sun and Moon Colors/////|
+
+screen.Sun_and_Moon_Colors = Цвет Солнца/Луны
+
+[-----sunPathRotation-----]
+
+option.sunPathRotation = Наклон Солнца
+
+[-----sun illuminance-----]
+
+option.sun_illuminance = Яркость Солнца
+
+[-----moon illuminance-----]
+
+option.moon_illuminance = Яркость Луны
+
+[-----Moonphase Based Moonlight-----]
+
+option.MOONPHASE_BASED_MOONLIGHT = Уровни яркости лунных фаз
+
+{#####_Settings For Supported Mods_#####}
+
+screen.Mod_support = Настройка Для Поддерживаемых Модов
+
+[-----ResourcePack Sky-----]
+
+option.RESOURCEPACK_SKY = Небо РесурсПака
+    value.RESOURCEPACK_SKY.0 = Не использовать
+    value.RESOURCEPACK_SKY.1 = Небо + Шейдерное солнце/луна
+    value.RESOURCEPACK_SKY.2 = Полностю небо
+    value.RESOURCEPACK_SKY.2 = Шейдерное Небо + РП (RP) солнце/луна
+
+|/////Waving Stuff/////|
+
+screen.Waving_Stuff = Развевающиеся Вещи
+    
+[-----Waving Plants-----]
+
+    option.WAVY_PLANTS = Развевающиеся Растения
+
+[-----Waving Strength-----]
+
+    option.WAVY_STRENGTH = Сила Развевания
+
+[-----Waving Speed-----]
+
+    option.WAVY_SPEED = Скорость Развевания
+
+{#####_Direct Light_#####}
+
+screen.Direct_Light = Прямой Свет
+
+|/////Shadows/////|
+
     screen.Shadows = Тени
-        option.SCREENSPACE_CONTACT_SHADOWS = Тени от объектов
-        option.RENDER_ENTITY_SHADOWS = Тени от сущностей
-        option.shadowMapResolution = Разрешение теней
-        option.shadowDistance = Дистанция теней 
-            value.shadowDistance.64.0 = 4 Чанков
+
+[-----Colored Shadows-----]
+
+        option.TRANSLUCENT_COLORED_SHADOWS = Цветные Тени
+
+[-----Contact Shadows-----]
+
+        option.SCREENSPACE_CONTACT_SHADOWS = В пространстве-экрана Тени
```

Comment:
```suggestion
        option.SCREENSPACE_CONTACT_SHADOWS = Тени в пространстве-экрана 
```

Reply:
It's my 100% mistake.
Thanks for noticing!